internal/keypairs/rsa.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // MinimumRSABitLength is the minimum allowed bit-length when generating RSA keys.
-const MinimumRSABitLength int = 12
```

Comment:
This and other RSA changes were required to get unit tests passing... I believe this is due to the update to go 1.24